File: python/6kyu/Multiples_of_3_or_5.py
def solution(number):
    number_list = []
    i = 3
    while i < number:
        if i < 3: return 0
        elif i % 3 == 0 and i % 5 == 0: number_list.append(i)
        elif i % 3 == 0 or i % 5 == 0: number_list.append(i)
        i += 1

    return sum(number_list)
  

File: typescript/6kyu/Multiples_of_3_or_5.ts
export class Challenge {
  static solution(number: number) {
    if (number <= 3) return 0;
    const numberArr: number[] = [];
    for (let i = 3; i < number; i++) {
      if (i % 3 === 0 && i % 5 === 0) {
        numberArr.push(i);
      } else if (i % 3 === 0) {
        numberArr.push(i);
      } else if (i % 5 === 0) {
        numberArr.push(i);
      }
    }
    return numberArr.reduce((a, b) => a + b);
  }
}
